README.md

```diff
@@ -45,4 +45,23 @@ allow {
     input.method == "GET"
     input.user == "bob"
 }
-```
\ No newline at end of file
+```
+
+## Trying the example
+- As a manager, create a car (this should be allowed):
+```
+curl -H 'Authorization: alice' -H 'Content-Type: application/json' \
```

Comment:
`Authorization: alice` doesn't look right. You're probably looking to include the `user` as part of the input, as that is what your policy is looking for.

Reply:
Your code is using this way to include the user to input: https://github.com/open-policy-agent/example-api-authz-go/blob/2b56096cb0fa68f6550b66d0d68a0250ee904ebc/internal/api/api.go#L216

Reply:
Oh, I should probably have checked the code for that handler 😄 That's really not a nice way to use the HTTP spec, but I guess we can live with that for now then.